src/renderer/components/layout/PrimaryLayout/Wallets/WalletMenu.tsx

```diff
@@ -102,11 +102,17 @@ const WalletMenu = ({ children, chains, wallets }: PropsWithChildren<Props>) =>
               <section className={cn('relative w-[289px] bg-card-background')}>
                 <header className="px-5 py-3 flex items-center justify-between border-b border-divider">
                   <SmallTitleText>{t('wallets.title')}</SmallTitleText>
-                  <DropdownButton
-                    options={DropdownOptions}
-                    className="w-[134px] py-2 h-8.5"
-                    title={t('wallets.addButtonTitle')}
-                  />
+
+                  <ButtonDropdown className="w-[134px] py-2 h-8.5" title={t('wallets.addButtonTitle')}>
```

Comment:
From now on `ButtonDropdown` is a button with container for list elements
List elements should be chosen outside of `ButtonDropdown`

Reply:
Maybe we should include button inside `ButtonDropdown.Item`? Since it's a dropdown menu we gonna have button element every time and might as well include in `ButtonDropdown.Item` and pass children like
```
<>
    <Icon name={icon as IconNames} size={20} className="text-icon-accent" />
    <FootnoteText className="text-text-secondary">{title}</FootnoteText>
</>
```